packages/ckeditor5-ui/src/panel/balloon/balloonpanelview.ts

```diff
@@ -245,7 +248,8 @@ export default class BalloonPanelView extends View {
 				defaultPositions.northArrowSouthMiddleEast,
 				defaultPositions.northArrowSouthWest,
 				defaultPositions.northArrowSouthEast,
-				defaultPositions.viewportStickyNorth
+				defaultPositions.viewportStickyNorth,
+				defaultPositions.viewportHidden
```

Comment:
See my comment in image/ui/utils.ts. I would try injecting this position in L257 whether there were no positions specified by the user (defaults being used) or they used some specific ones (even custom ones).